### PR TITLE
Refine all the docker-compose files (#96)

### PR DESCRIPTION
**Pull request by Aymerix01**
The issue was that you should write docker-compose -f docker-compose.dev.mongo.yml up -d node --build, and then go to frontend (cd frontend) and launch the dev interface with yarn dev:start HTTP=false to be able to see something on http://localhost3000/ . Now, you only need to launch docker-compose up -d and go to http://localhost to see TDrive. 
TDrive refine docker

Everything works except OnlyOffice.
OnlyOffice error

Important to note, the only docker-compose file used is docker-compose.yml but i let the others in case someone needs them.

### CODE REVIEW

**Comment by Aymerix01** [comment]
There may be a problem on Windows due to a bad convert line break on entrypoint.sh and self-signed.sh .
This error leads to an instant stop of the tdrive-frontend contener on docker and with a log in the container like this: exec /entrypoint.sh: no such file or directory (or /self-signed.sh: no such file or directory )
I solved it with Notepad++ (Open entrypoint.sh with notepad++ then edit, EOL conversion and switch to Unix (LF) )
Notepadd++ convert line break

File: tdrive/docker/onlyOffice-connector/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Base image
+FROM docker.io/onlyoffice/documentserver
```
Comment (chibenwa) [inline tdrive/docker/onlyOffice-connector/Dockerfile]: Please use a fixed version if available

**Review by chibenwa — COMMENTED** [review]
I would also add a friendly README in `tdrive/docker` either:

 - ... Pointing to the doc `run tdrive` if available
 - ... or documenting itself the way to run tdrive with docker-compose...

File: tdrive/docker/onlyOffice-connector/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Base image
```
Comment (shepilov) [inline tdrive/docker/onlyOffice-connector/Dockerfile]: why do you need a connector docker file here?
it should be only in it's repository, and you can use existing image

File: tdrive/docker/tdrive-frontend/Dockerfile [hunk 3]
```diff
@@ -20,6 +20,7 @@ COPY docker/tdrive-frontend/nginx.conf /etc/nginx/nginx.conf
 COPY docker/tdrive-frontend/site.conf /etc/nginx/sites-available/site.template
 COPY docker/tdrive-frontend/redirect.conf /etc/nginx/sites-available/redirect
 COPY docker/tdrive-frontend/entrypoint.sh /
+COPY docker/tdrive-frontend/self-signed.sh /
```
Comment (shepilov) [inline tdrive/docker/tdrive-frontend/Dockerfile:23]: and this one also, why do you need it in root folder?

**Pull request closed without merging**